specification/security/resource-manager/Microsoft.Security/preview/2021-01-01-preview/examples/secureScores/ListSecureScoreControls_managementGroup_example.json

```diff
@@ -0,0 +1,218 @@
+{
+  "parameters": {
+    "api-version": "2021-01-01-preview",
+    "scope": "providers/Microsoft.Management/managementGroups/groupId"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Management/managementGroups/groupId/providers/Microsoft.Security/secureScores/ascScore/secureScoreControls/a9909064-42b4-4d34-8143-275477afe18b",
```

Comment:
same, remove /secureScores from id and type

Reply:
the purpose of the top-level get is to expose all the child resources across all secureScores in a management group